vcfa/config_test.go

```diff
@@ -0,0 +1,1043 @@
+	setBoolFlag(&vcfaShowCount, "vcfa-show-count", "VCFA_SHOW_COUNT", "Show number of pass/fail tests")
+	setBoolFlag(&vcfaReRunFailed, "vcfa-re-run-failed", "VCFA_RE_RUN_FAILED", "Run only tests that failed in a previous run")
+	setBoolFlag(&testDistributedNetworks, "vcfa-test-distributed", "", "enables testing of distributed network")
+	setBoolFlag(&enableDebug, "vcfa-debug", "GOVCD_DEBUG", "enables debug output")
+	setBoolFlag(&vcfaTestVerbose, "vcfa-verbose", "TEST_VERBOSE", "enables verbose output")
+	setBoolFlag(&enableTrace, "vcfa-trace", "GOVCD_TRACE", "enables function calls tracing")
+	setBoolFlag(&vcfaShortTest, "vcfa-short", "VCFA_SHORT_TEST", "runs short test")
+	setBoolFlag(&vcfaAddProvider, "vcfa-add-provider", envVcfaAddProvider, "add provider to test scripts")
+	setBoolFlag(&vcfaSkipTemplateWriting, "vcfa-skip-template-write", envVcfaSkipTemplateWriting, "Skip writing templates to file")
+	setBoolFlag(&vcfaRemoveOrgVdcFromTemplate, "vcfa-remove-org-vdc-from-template", envVcfaRemoveOrgVdcFromTemplate, "Remove org and VDC from template")
+	setBoolFlag(&vcfaTestOrgUser, "vcfa-test-org-user", envVcfaTestOrgUser, "Run tests with org user")
+	setStringFlag(&vcfaSkipPattern, "vcfa-skip-pattern", "VCFA_SKIP_PATTERN", "Skip tests that match the pattern (implies vcfa-pre-post-checks")
+	setBoolFlag(&skipLeftoversRemoval, "vcfa-skip-leftovers-removal", "VCFA_SKIP_LEFTOVERS_REMOVAL", "Do not attempt removal of leftovers at the end of the test suite")
+	setBoolFlag(&silentLeftoversRemoval, "vcfa-silent-leftovers-removal", "VCFA_SILENT_LEFTOVERS_REMOVAL", "Omit details during removal of leftovers")
+	setStringFlag(&testListFileName, "vcfa-partition-tests-file", "VCFA_PARTITION_TESTS_FILE", "Name of the file containing the tests to run in the current partition node")
+	setIntFlag(&numberOfPartitions, "vcfa-partitions", "VCFA_PARTITIONS", "")
+	setIntFlag(&partitionNode, "vcfa-partition-node", "VCFA_PARTITION_NODE", "")
+}
+
+// Structure to get info from a config json file that the user specifies
+type TestConfig struct {
+	Provider struct {
+		User                    string `json:"user"`
+		Password                string `json:"password"`
+		Token                   string `json:"token,omitempty"`
+		ApiToken                string `json:"api_token,omitempty"`
+		ApiTokenFile            string `json:"api_token_file,omitempty"`
+		ServiceAccountTokenFile string `json:"service_account_token_file,omitempty"`
+
+		// Tenant Manager version and API version, they allow tests to
+		// check for compatibility without using an extra connection
+		TmVersion    string `json:"tmVersion,omitempty"`
+		TmApiVersion string `json:"tmApiVersion,omitempty"`
+
+		Url                      string `json:"url"`
+		SysOrg                   string `json:"sysOrg"`
+		AllowInsecure            bool   `json:"allowInsecure"`
+		TerraformAcceptanceTests bool   `json:"tfAcceptanceTests"`
+		UseVcdConnectionCache    bool   `json:"useVcdConnectionCache"`
```

Comment:
`Vcd`. Perhaps it could be just `UseConnectionCache`

Reply:
Fixed